### PR TITLE
en,zh: add s3 fields description for br doc

### LINKED ISSUE #1391: add `.spec.s3.endpoint` for BR doc
## Change Request

**This repository is ONLY used to solve issues related to TiDB Operator documentation.
For other issues (related to TiDB, PD, etc), please move to [other repositories](https://github.com/pingcap/).**

Please answer the following questions before submitting your issue. Thanks!

1. Describe what you find is inappropriate or missing in the existing docs.

`.spec.s3.endpoint` in https://docs.pingcap.com/zh/tidb-in-kubernetes/stable/backup-restore-overview#s3-%E5%AD%98%E5%82%A8%E5%AD%97%E6%AE%B5%E4%BB%8B%E7%BB%8D

2. Describe your suggestion or addition.

3. Provide some reference materials (documents, websites, etc) if you could.

### PR DESCRIPTION
**Pull request by just1900**
### What is changed, added, or deleted? (Required)
closes #1391 

### Which TiDB Operator version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v1.3 (TiDB Operator 1.3 versions)
- [x] v1.2 (TiDB Operator 1.2 versions)
- [ ] v1.1 (TiDB Operator 1.1 versions)
- [ ] v1.0 (TiDB Operator 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: 
- Other reference link(s):

### CODE REVIEW

File: zh/backup-restore-cr.md [hunk 1]
```diff
@@ -115,6 +115,8 @@ summary: 介绍用于备份与恢复的 Custom Resource (CR) 资源的各字段
 * `.spec.s3.region`：使用 Amazon S3 存储备份，需要配置 Amazon S3 所在的 region。
 * `.spec.s3.bucket`：兼容 S3 存储的 bucket 名字。
 * `.spec.s3.prefix`：如果设置了这个字段，则会使用这个字段来拼接在远端存储的存储路径 `s3://${.spec.s3.bucket}/${.spec.s3.prefix}/backupName`。
+* `.spec.s3.endpoint`：兼容 S3 的存储服务 endpoint。
```
Comment (csuzhangxc) [inline zh/backup-restore-cr.md:118]: 举个例子吗

Comment (just1900) [reply zh/backup-restore-cr.md:118]: 是说加一个使用 endpoint 的 backup 还是像这样？
```suggestion
* `.spec.s3.endpoint`：兼容 S3 的存储服务 endpoint，例如 http://minio.minio.svc.cluster.local:9000
```

Comment (csuzhangxc) [reply zh/backup-restore-cr.md:118]: 就是你 suggested 这样

File: zh/backup-restore-cr.md [hunk 2]
```diff
@@ -115,6 +115,8 @@ summary: 介绍用于备份与恢复的 Custom Resource (CR) 资源的各字段
 * `.spec.s3.region`：使用 Amazon S3 存储备份，需要配置 Amazon S3 所在的 region。
 * `.spec.s3.bucket`：兼容 S3 存储的 bucket 名字。
 * `.spec.s3.prefix`：如果设置了这个字段，则会使用这个字段来拼接在远端存储的存储路径 `s3://${.spec.s3.bucket}/${.spec.s3.prefix}/backupName`。
+* `.spec.s3.endpoint`：兼容 S3 的存储服务 endpoint，例如 `http://minio.minio.svc.cluster.local:9000`。
+* `.spec.s3.secretName`：指定访问兼容 S3 存储的密钥信息的 Secret 名称。
```
Comment (csuzhangxc) [inline zh/backup-restore-cr.md:119]: 要不要再详细说下需要 `access_key` 与 `secret_key` 这两个 data  😂

File: zh/backup-restore-cr.md [hunk 3]
```diff
@@ -115,6 +115,8 @@ summary: 介绍用于备份与恢复的 Custom Resource (CR) 资源的各字段
 * `.spec.s3.region`：使用 Amazon S3 存储备份，需要配置 Amazon S3 所在的 region。
 * `.spec.s3.bucket`：兼容 S3 存储的 bucket 名字。
 * `.spec.s3.prefix`：如果设置了这个字段，则会使用这个字段来拼接在远端存储的存储路径 `s3://${.spec.s3.bucket}/${.spec.s3.prefix}/backupName`。
+* `.spec.s3.endpoint`：兼容 S3 的存储服务 endpoint，例如 `http://minio.minio.svc.cluster.local:9000`。
+* `.spec.s3.secretName`：指定访问兼容 S3 存储的密钥信息（包含 access key 和 secret key）的 Secret 名称。
```
Comment (ran-huang) [inline zh/backup-restore-cr.md:119]:
```suggestion
* `.spec.s3.secretName`：访问兼容 S3 存储的密钥信息（包含 access key 和 secret key）的 Secret 名称。
```

**Review by ran-huang — COMMENTED** [review]
Rest LGTM

**Review by csuzhangxc — APPROVED** [review]

**Comment by ran-huang** [comment]
@just1900 Do they apply to 1.2 as well?

**Comment by just1900** [comment]
> @just1900 Do they apply to 1.2 as well?

yes, have added `cherry-pick-1.2` label

**Review by ran-huang — APPROVED** [review]

**Pull request merged**